src/application/tabs.css

```diff
@@ -116,13 +113,21 @@
   margin-left: 0;
 }
 
+/* .p-mod-current state */
+
 .p-TabBar-tab.p-mod-current {
-    color: var(--jp-ui-font-color0);
+  background: var(--jp-layout-color1);
+}
+
+
+.p-TabBar-tab.p-mod-current {
+  color: var(--jp-ui-font-color0);
 }
 
 .p-TabBar.p-mod-top .p-TabBar-tab.p-mod-current,
 .p-TabBar.p-mod-bottom .p-TabBar-tab.p-mod-current {
-  min-height: calc(var(--jp-private-horizontal-tab-height) - var(--jp-border-width));
+  overflow: visible;
```

Comment:
The overflow rule should probably be set the same for all tabs, not just the current tab.